cinema/views.py

```diff
@@ -81,12 +109,14 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         MovieSession.objects.all()
         .select_related("movie", "cinema_hall")
         .annotate(
-            tickets_available=F("cinema_hall__rows")
-            * F("cinema_hall__seats_in_row")
-            - Count("tickets")
+            tickets_available=F("cinema_hall__rows") # noqa
+                              * F("cinema_hall__seats_in_row") # noqa
+                              - Count("tickets") # noqa
```

Comment:
The `# noqa` comments are used to suppress linter warnings. Ensure that these are necessary and that the code is functioning as expected. If the linter warnings are valid, consider refactoring the code to address them instead of suppressing them.